plugins/org.eclipse.reddeer.eclipse/META-INF/MANIFEST.MF

```diff
@@ -98,3 +98,4 @@ Export-Package: org.eclipse.reddeer.eclipse,
  org.eclipse.reddeer.eclipse.debug.ui.views.variables,
  org.eclipse.reddeer.eclipse.topmenu,
  org.eclipse.reddeer.eclipse.epp.logging.aeri.ide.dialogs
+Import-Package: org.jboss.reddeer.swt.impl.menu
```

Comment:
also please do not use Import-Package but Require-Bundle. 
Require-Bundle already contains org.eclipse.reddeer.swt so there is no need for this line at all.